DQM/L1TMonitor/src/L1TStage2Shower.cc

```diff
@@ -97,7 +97,7 @@ void L1TStage2Shower::analyze(const edm::Event& e, const edm::EventSetup& c) {
     int ring = detId.ring();
     int chamber = detId.chamber();
     int sr = histIndexCSC.at({station, ring});
-    if (endcap == -1)
+    if (endcap == 1)
```

Comment:
By the way, since `endcap` is only used to check whether it is equal 1 or not, a further simplification could be replacing L95 with
```C++
    int endcap = detId.endcap();
```